WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -72,10 +72,16 @@ final class OrderDetailsViewModel {
         }
     }
 
+    /// IPP Configuration loader
+    private lazy var configurationLoader = {
+        CardPresentConfigurationLoader(stores: stores)
+    }()
```

Comment:
nit: can also be one line 😄 
```suggestion
    private lazy var configurationLoader = CardPresentConfigurationLoader(stores: stores)
```